server/sonar-web/src/main/js/apps/users/groups-view.js

```diff
@@ -9,23 +9,24 @@ define([
     itemTemplate: Templates['users-group'],
 
     onRender: function () {
+      console.log(this.model);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=javascript%3AS2228)